pkg/version/version_unix.go

```diff
@@ -18,23 +18,31 @@ import (
 
 func parseKernelVersion(ver string) (semver.Version, error) {
 	verStrs := strings.Split(ver, ".")
-	switch {
-	case len(verStrs) < 2:
+
+	if len(verStrs) < 1 {
```

Comment:
Should logic be included to check if `verStrs` exceeds the expected maximum length?